docker/conda/environments/cuda11.5_dev.yml

```
@@ -60,6 +62,7 @@ dependencies:
     - mlflow>1.29,<2
     - mrc=23.01
     - myst-parser==0.17
+    - nbsphinx=0.8.10 # For Sphinx documentation
```

If we are moving the dependencies needed to build the docs into the main conda env file, we should be sure we add all of them and then rm the `docs/requirements.txt` and `docs/conda_docs.yml` files, and update `ci/scripts/github/docs.sh` script.
```suggestion
    - nbsphinx=0.8.10 # For Sphinx documentation
```

---

This was the wrong choice on my part. I'll roll these changes back.